third_party/wabt/include/wabt/interp/jit/jit-imath64-inl.h

```diff
@@ -413,6 +413,54 @@ static bool emitCompare(sljit_compiler* compiler, Instruction* instr) {
       opcode = SLJIT_SUB | SLJIT_SET_GREATER_EQUAL;
       type = SLJIT_GREATER_EQUAL;
       break;
+    case Opcode::F32Eq:
+      opcode = SLJIT_CMP_F32;
+      type = SLJIT_F_EQUAL;
```

Comment:
These are undefined for nan comparisons. I suspect wasm defines the nan results, so please use those forms.